docs(dialog): enhance refresh method description (#1458)
telerik/blazor-docs: components/dialog/overview.md

| Method | Description |
| --- | --- |

> | `Refresh` | The way the Telerik Dialog component is displayed is by rendering it as a child of the `TelerikRootComponent`, rather than its initial declaration location. As a result, when the Dialog is shown and its content is updated, it doesn't refresh automatically. To ensure that the updated content is correctly displayed within the Dialog, the `Refresh` method must be manually invoked. |

Reviewer: I would start with a short and simple sentence what the method does, and then provide the additional insights.

> | `Refresh` | Use the method to programmatically re-render the Dialog. <br /> The Dialog is rendered as a child of the `TelerikRootComponent`, instead of where it was originally declared. As a result, it doesn't automatically refresh when its content is updated. In such cases, the `Refresh` method comes in handy as it can be manually invoked to ensure that the updated Dialog content is displayed correctly. |

Reviewer: ```suggestion
| `Refresh` | Re-renders the Dialog. <br /> The Dialog is rendered as a child of the `TelerikRootComponent`, instead of where it is declared. As a result, it doesn't automatically refresh when its content is updated. In such cases, the `Refresh` method comes in handy to ensure that the Dialog content is up-to-date. |
```
